rpcserverhandlers_test.go

```diff
@@ -1919,6 +1924,223 @@ func TestHandleGetBestBlockHash(t *testing.T) {
 	}})
 }
 
+func TestHandleGetBlockchainInfo(t *testing.T) {
+	hash, _ := chainhash.NewHashFromStr("00000000000000001e6ec1501c858506de1de47" +
+		"03d1be8bab4061126e8f61480")
+	prevHash, _ := chainhash.NewHashFromStr("00000000000000001a1ec2becd0dd90bfbd" +
+		"0c65f42fdaf608dd9ceac2a3aee1d")
+	genesisHash, _ := chainhash.NewHashFromStr("298e5cc3d985bfe7f81dc135f360abe0" +
+		"89edd4396b86d2de66b0cef42b21d980")
+	genesisPrevHash, _ := chainhash.NewHashFromStr("0000000000000000000000000000" +
+		"000000000000000000000000000000000000")
+	testChainParams := &chaincfg.Params{
```

Comment:
This is fine, but you might want to steal `cloneParams` from `blockchain` and use that.  Then you can just modify the `Deployments` specifically.  That approach is much harder to break when the Params struct is updated.

Reply:
I explicitly defined the `Name`, `Deployments`, and `PowLimitBits` params here since those are the only ones that `handleGetBlockchainInfo` depends on, and this would prevent the tests from breaking if the values of any of those change.

I still added in the `cloneParams` as you suggested, though in this case, I think it would only help if a new field were added to the Params struct that `handleGetBlockchainInfo` uses internally but doesn't impact the output of `handleGetBlockchainInfo` (otherwise this test could need to be updated anyway).